site/.env.production

```diff
@@ -0,0 +1 @@
+GATSBY_SHOW_DEPRECATED_WARNING=FALSE
```

Comment:
We need to change this to TRUE when we release 2.0.0. Or to be more specific, when we build the old site and serve it under different folder.